content/en/tracing/guide/latency_investigator.md

```diff
@@ -0,0 +1,120 @@
+- **Narrow the scope**: Isolate issues to specific data centers, clusters, or user segments with [Tag Analysis][1].
+- **Find root causes**: Detect faulty deployments, database slowness, third-party service failures, infrastructure problems, and service-level issues.
+
+## Starting an investigation
+
+Launch investigations from APM service pages or resource pages.
+
+To begin:
+
+1. Navigate to a service showing latency issues.
+2. Find the latency graph showing the anomaly.
+3. Hover over the graph and click **Investigate**.
+
+{{< img src="tracing/guide/apm_investigator/apm_investigator_entrypoint.png" alt="APM Investigator entrypoint" style="width:90%;">}}
+
+This opens the investigation side panel.
+
+## Investigation workflow
+
+### Define context: Select slow and normal spans
+
+To trigger the latency analysis, select two zones on the point plot:
+
+- **Slow**: Problematic, slow spans
+- **Normal**: Baseline, healthy spans
+
+<div class="alert alert-info">Latency anomalies detected by Watchdog are pre-selected.</div>
+
+{{< img src="tracing/guide/apm_investigator/latency_selection.png" alt="Selection of slow spans on the point plot" style="width:90%;">}}
+
+This comparison between the slow and normal spans drives all subsequent analysis.
+
+### Step 1: Identify the latency bottleneck
+
+The investigator identifies whether latency originates from your service or its downstream dependencies (services, databases, third-party APIs).
+
+**Analysis approach**:
+The investigator compares trace data from both your selected slow and normal periods. To find the service responsible for the latency increase, it compares:
+
+- **Execution Time** (self-time; the time spent on the service that is not spent in downstream dependencies) spent within each service across the two datasets. The service with the largest absolute latency increase is the primary focus.
+- **Call Patterns Between Services**: It analyzes changes in the number of requests between services. For example, if service Y significantly increases its calls to downstream service X, the investigator might identify Y as the bottleneck.
+
+Based on this comprehensive analysis, the investigator recommends a service as the likely latency bottleneck. Expand the latency bottleneck section to see details about the comparison between slow and normal traces. A table surfaces the changes in self-time and in the number of inbound requests by service.
+
+The following example shows two side-by-side flame graphs that compare slow traces against healthy traces in more detail. Use the arrows to cycle through example traces and click **View** to open the trace in a full pageview.
+
+{{< img src="tracing/guide/apm_investigator/latency_bottleneck.png" alt="Latency bottleneck section" style="width:100%;">}}
+
+From the table, click the `+` icon that appears when you hover over a row to add that service to the context of investigation for further analysis of recent changes on the service.
```

Comment:
```suggestion
To investigate recent changes to a service, click the `+` icon that appears when you hover over a row to add it as context for your investigation.
```
I'm not 100% sure this is accurate - just an idea to make the sentence a little easier to follow 🙂